src/Interfaces/IGetQueueAvailabilityOptionalParams.ts

```diff
@@ -0,0 +1,16 @@
+import InitContext from "../Model/InitContext";
+
+export default interface IGetQueueAvailabilityOptionalParams {
+  /**
+   * JWT token from the portal
+   */
+  authenticatedUserToken?: string;
+  /**
+   * Whether to get init context from browser.
+   */
+  getContext?: boolean;
```

Comment:
Is `getContext` needed?

Reply:
Yes, It is required to update the engagement context information